lib/src/solver/incompatibility.dart

```diff
@@ -223,25 +222,24 @@ class Incompatibility {
         .map((term) => _terse(term, details))
         .join(' or ');
 
+    var buffer =
+        new StringBuffer("${_terse(thisPositive, details, allowEvery: true)} ");
```

Comment:
The interpolation isn't buying you much. Maybe just:

```dart
new StringBuffer(_terse(thisPositive, details, allowEvery: true) + " ");
```

Reply:
Done.